examples/examples/fetch_and_cat.rs

```diff
@@ -55,7 +55,13 @@ fn main() {
 
         eprintln!();
 
-        let stream = ipfs.cat_unixfs(cid, None);
+        let stream = match ipfs.cat_unixfs(cid, None).await {
+            Ok(stream) => stream,
+            Err(e) => {
+                eprintln!("Error: {}", e);
+                exit(1);
+            }
+        };
```

Comment:
nit: this could probably be just
```
let stream = ipfs.cat_unixfs(cid, None).await.unwrap_or_else(|e| {
    eprintln!("Error: {}", e);
    exit(1);
});
```